src/Builder/AsyncBuilderTrait.php

```diff
@@ -33,10 +36,22 @@ public function generateAsync(): string
             'sensiolabs_gotenberg.builder' => $this::class,
         ]);
 
+        $url = $this->webhookUrl ?? null;
+        $errorUrl = $this->errorWebhookUrl ?? null;
+        if (!$url || !$errorUrl) {
+            if (!($this->urlGenerator ?? null)) {
+                throw new \LogicException(\sprintf('A webhook URL or Router is required to use "%s" method. Set the URL or try to run "composer require symfony/routing".', __METHOD__));
```

Comment:
we may throw if we are missing an error url whereas it should fallback to `$url`